applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -87,7 +87,7 @@ static bool bitrate_check(const struct bt_codec *codec)
 
 static bool adv_data_parse(struct bt_data *data, void *user_data)
 {
-	if (data->type == BT_DATA_NAME_COMPLETE && data->data_len == DEVICE_NAME_PEER_LEN) {
+	if (data->type == 0x30 && data->data_len) {
```

Comment:
Where is this magic number from?

Reply:
It's the broadcast name number defined here: https://www.bluetooth.com/wp-content/uploads/2022/11/assigned_numbers_release.pdf, it hasn't been defined in host yet, but there is a PR coming for it

Reply:
If the PR is likely to be merged soon, then I don't see anything wrong with adding it as `[nrf fromtree]` and then using it here. Please update me on the PR.